### PR TITLE
Add a .graphml parser

### PR DESCRIPTION
**Pull request by SmalRat**
BEGINRELEASENOTES

Added a basic program to read graphs in the .graphml format

ENDRELEASENOTES

### CODE REVIEW

File: data/sequencer_demo/df_sequencer_demo.graphml
Comment (hegner) [inline data/sequencer_demo/df_sequencer_demo.graphml:1]: @m-fila - can you fix the output format of your tool?

**Review by hegner — APPROVED** [review]
Thanks!
There are two things:
1) can you open an issue for the additions of tests
2) can you prepare a PR at https://github.com/zhangliye/GraphMLReader.jl to include your fixes?

File: data/sequencer_demo/df_sequencer_demo.graphml
Comment (m-fila) [reply data/sequencer_demo/df_sequencer_demo.graphml:1]: Thanks for pointing out and sorry about the confusion. Corrected in #3 There should be only one graph in each file

**Comment by SmalRat** [comment]
> Thanks! There are two things:
> 
> 1. can you open an issue for the additions of tests
> 2. can you prepare a PR at https://github.com/zhangliye/GraphMLReader.jl to include your fixes?

Will do

**Comment by SmalRat** [comment]
Made a PR and opened an issue.

**Pull request merged**